### PR TITLE
Install additional gems

### PR DESCRIPTION
**Pull request by okkez**
Users can install additional gems using Gemfile.local and/or
plugins/*/{Gemfile,PluginGemfile}.

### CODE REVIEW

File: 3.1/docker-entrypoint.sh [hunk 1]
```diff
@@ -101,6 +101,8 @@ case "$1" in
 		
 		# ensure the right database adapter is active in the Gemfile.lock
 		cp "Gemfile.lock.${adapter}" Gemfile.lock
+                # ensure to install additional gems for Gemfile.local and plugins
+                bundle echeck || bundle install --without development test
```
Comment (tianon) [inline 3.1/docker-entrypoint.sh]: I can't find any reference to `bundle echeck`, but I did find `bundle check` (https://bundler.io/v1.15/man/bundle-check.1.html); was this perhaps a typo?

This seems generally sane, given that we no longer invoke `bundle install` (#80), and given that it's conditional on whether it's actually deemed necessary.

I think these are also currently indented with spaces, and need to be indented with tabs instead.

Comment (okkez) [inline 3.1/docker-entrypoint.sh]: Oh, this is typo. I've fixed and force pushed.

**Comment by tianon** [comment]
Thanks!

**Comment by zx1986** [comment]
It seems Gemfile.local or plugins/*/{Gemfile,PluginGemfile} not work for me :-(

Here is my docker-compose.yml:

```
projects:
  image: redmine:3-passenger
  container_name: projects
  restart: always
  ports:
    - "3000:3000"
  volumes:
    - ./config/database.yml:/usr/src/redmine/config/database.yml
    - ./config/configuration.yml:/usr/src/redmine/config/configuration.yml
    - ./public/themes:/usr/src/redmine/public/themes
    - ./plugins:/usr/src/redmine/plugins
    - ./files:/usr/src/redmine/files
    - ./log:/usr/src/redmine/log
  env_file:
    - docker.env
```

Where should I put my Gemfile.local into?

**Comment by zx1986** [comment]
https://github.com/docker-library/redmine/blob/master/3.4/docker-entrypoint.sh#L103

I think I should mount a Gemfile.lock.production into container?
But it didn't make sense ...
How about these plugins/*/Gemfile?

**Comment by zx1986** [comment]
And I didn't get this:
https://github.com/docker-library/redmine/blob/master/3.4/Dockerfile#L79

Seems Gemfile.lock.${adapter} is copy from Gemfile.lock (which is the Redmine's original Gemfile.lock), 
so how should I add my own / plugins gems into that?

**Pull request merged**